css/css.go

```diff
@@ -1201,6 +1201,41 @@ func (c *cssMinifier) minifyProperty(prop Hash, values []Token) []Token {
 			values[0].Data = oneBytes
 			values[0].Ident = 0
 		}
+	case UnicodeRange:
+		// U+2600-26FF → U+26??
+		for i, value := range values {
+			// Starts with "U+..." or "u+..."
+			if len(value.Data) >= 2 && value.Data[0]|32 == 'u' && value.Data[1] == '+' {
```

Comment:
This is a personal style I have, but I prefer to write it the other way around, as in `1 < len(value.Data)` or `2 <= len(value.Data)`. That way the left side is always smaller then the right side, which is more intuitive (e.g. when we write intervals). It's easier to process mentally, for me at least.